
sys/signal.h does almost the same in both glibc and musl: it includes "signal.h"

However with musl there is also a warning macro about this, which is interpreted as
an error during building.

Fixes error:

| In file included from ../../sources/xrdp-0.10.4.1/waitforx/waitforx.c:5:
| <...>/usr/include/sys/signal.h:1:2: error: #warning redirecting incorrect #include <sys/signal.h> to <signal.h> [-Werror=cpp]
|     1 | #warning redirecting incorrect #include <sys/signal.h> to <signal.h>
|       |  ^~~~~~~
| cc1: all warnings being treated as errors

Upstream-Status: Submitted [https://github.com/neutrinolabs/xrdp/pull/3678]
Signed-off-by: Gyorgy Sarvari <skandigraun@gmail.com>

--- ./waitforx/waitforx.c.orig	2025-11-25 14:38:46.464337398 +0100
+++ ./waitforx/waitforx.c	2025-11-25 14:38:52.007441093 +0100
@@ -2,7 +2,6 @@
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <sys/signal.h>
 #include <unistd.h>
 
 #include "config_ac.h"
